project_resource_calendar/models/calendar_event.py

```diff
@@ -305,3 +305,15 @@ def validate_client_id_write(self, vals):
             if 'partner_ids' in vals and self.client_id.id not in vals['partner_ids'][0][2]:
                 vals['partner_ids'] = [
                     (6, 0, vals['partner_ids'][0][2] + [self.client_id.id])]
+
+    @api.multi
+    def unlink(self):
+        if self.event_task_id.task_state in \
```

Comment:
if it is api.multi use the for rec in self:

Reply:
@jananjoy keep only this unlink, we dont need the other unlink for the moment. We need to prevent the delete of the clone only

Reply:
and add the equivalent test